README.md

```diff
@@ -79,6 +79,19 @@ config :mix_test_watch,
   clear: true
 ```
 
+## Excluding files or directories
+
+To ignore changes from specific files or directories just add `exclude:` patterns
+to your config in `mix.exs`:
+
+```elixir
+config :mix_test_watch,
+  exclude: ["db_migration/.*", "useless_.*\.exs"]
+```
+
+It accepts either regexp struct `~r/\w+/` or pattern as string `"\w+"` which
```

Comment:
The pattern syntax commonly used in strings. You used it in your example in your previous reply.

The fact you used globs there in the string instead of regex reaffirms my belief that having regex strings would be confusing. If you have put either of those values in Dogma would crash.